examples/test_lra_value.cc

```diff
@@ -8,6 +8,7 @@ pre()
 {
     auto config = std::unique_ptr<SMTConfig>(new SMTConfig());
     const char* msg;
+    config->setOption(SMTConfig::o_produce_models, SMTOption(true), msg);
```

Comment:
Can we use `config->setProduceModels()` here as well?
It seems simpler than this.